src/api/rewrite.ts

```diff
@@ -0,0 +1,10 @@
+import { callSidecar } from './sidecar';
+
+export async function askForRewrite(selection: string): Promise<string> {
```

Comment:
Moved this function to the `/api` in order to have all the sidecar interaction in the api.